src/main/java/study/calculator/Calculator.java

```diff
@@ -0,0 +1,43 @@
+package study.calculator;
+
+public class Calculator {
+    private static final String DELIMITER = " ";
+    private Operator operator;
+
+    Calculator() {
+        operator = new Operator();
+    }
+
+    public Integer calculate(String expression) {
+        validate(expression);
+
+        String[] parts = expression.trim().split(DELIMITER);
+        if(parts.length < 3) {
```

Comment:
IntelliJ IDEA의 코드 자동 정렬 기능(⌥⌘L, Ctrl+Alt+L)을 사용하면 더 깔끔한 코드를 볼 수 있을 거예요.

```suggestion
        if (parts.length < 3) {
```

Reply:
앗, 이런 기본적인것도 체크 안하고 커밋하다니 부끄럽습니다 😊 알려주셔서 감사합니다 😃